docs/rule/no-negated-condition.md

```diff
@@ -9,9 +9,9 @@ It can be hard to reason about negated conditions:
 
 Negated conditions can often be avoided or simplified by:
 
-* Flipping `if (not condition) ... else ...` to `if .... else ...`
-* Replacing `if (not condition)` with `unless`
-* Replacing `unless (not condition)` with `if`
+* Flipping `if (not condition) { block1 } else { block2 }` to `if condition { block2 } else { block1 }`
```

Comment:
Can we make these examples use template syntax instead of JS? I find the intermixing of the JS and HBS a bit confusing

Reply:
Agreed, fixed!